.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+##### Issue type
+<!-- Pick one below and delete others -->
+ - Bug report
+ - Feature request
+ - Documentation issue
+ - Build issue
+ 
+##### Minetest version
+<!--
+Paste Minetest version between quotes below
```

Comment:
maybe ask for the output of `minetest --version`?

Reply:
done